README.md

```diff
@@ -38,6 +38,12 @@ or [zip format](https://artifacts.consensys.net/public/teku/raw/names/teku.zip/v
 We recommend only using release versions for MainNet, but `develop` builds are useful for testing 
 the latest changes on testnets.
 
+Release notifications are available via:
+* `teku` in [Consensys Discord](https://discord.gg/7hPv2T6), 
+* `client-updates` in [Ethstaker Discord](https://discord.gg/u2q7FRv)
```

Comment:
I probably wouldn't list this one as an official source. It is useful but we don't actually control it so can't do anything if they stop including our announcements there in the future.